client/src/components/RelatedItems/RelatedItems.jsx

```diff
@@ -81,14 +80,10 @@ function RelatedItems() {
       axios
         .all([getYOImages, getYODetails, getYOReviews])
         .then((response) => {
-          const aveRating = {
-            aveRating: _.reduce(response[2].data.results, (sum, num) => (
-              sum + num.rating
-            ), 0) / response[2].data.results.length
-          }
+          const ratingsArray = response[2].data.results;
```

Comment:
I like this way better! So much clearer than using plain old `response[2].data.results`